std_msgs/Header header
Observation[] obs
float64[6] rr
float64[6] dtr
float64 zwd